www/css/transition.css

```diff
@@ -0,0 +1,69 @@
+/* COMMENT OUT ALL THE SCALE PARTS FOR A SMOOTH HORIZONTAL ANIMATION */
```

Comment:
I guess you can delete this comment and the commented-out parts below?

Reply:
Ya ...  I will make this sure